src/Examples/Flowsheets/HDA_flowsheet_with_costing.ipynb

```diff
@@ -0,0 +1,1598 @@
+    "\n",
+    "\n",
+    "This reaction is often accompanied by an equilibrium side reaction\n",
+    "which forms diphenyl, which we will neglect for this example.\n",
+    "\n",
+    "This example is based on the 1967 AIChE Student Contest problem as\n",
+    "present by Douglas, J.M., Chemical  Design of Chemical Processes, 1988,\n",
+    "McGraw-Hill.\n",
+    "\n",
+    "Users may refer to the prior examples linked at the top of this notebook for detailed process descriptions of the two HDA configurations. As before, the properties required for this module are defined in\n",
+    "\n",
+    "- `hda_ideal_VLE.py`\n",
+    "- `idaes.models.properties.activity_coeff_models.BTX_activity_coeff_VLE`\n",
+    "- `hda_reaction_kinetic.py`\n",
+    "\n",
+    "Additionally, we will be importing externally-defined flowsheets for the two HDA configurations from\n",
+    "\n",
+    "- `hda_flowsheets.py`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import and run HDA Flowsheets\n",
+    "First, we will generate solved flowsheets for each HDA model. The external scripts build and set inputs for the flowsheets, initialize unit models and streams, and solve the flowsheets before returning the model objects. Note that the HDA flowsheets contain all unit models and stream connections, and no costing equations:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# source file for external flowsheets\n",
+    "from hda_flowsheets import hda_with_flash, hda_with_distillation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Building flowsheet...\n",
+      "\n",
+      "Setting inputs...\n",
+      "\n",
+      "Initializing flowsheet...\n",
+      "\n",
+      "Limiting Wegstein tear to 5 iterations to obtain initial solution, if not converged IPOPT will pick up and continue.\n",
+      "\n",
+      "WARNING: Wegstein failed to converge in 5 iterations\n",
+      "Solving flowsheet...\n",
+      "\n",
+      "Complete.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# alias 'm' for hda model with second flash unit\n",
+    "m = hda_with_flash(tee=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Building flowsheet...\n",
+      "\n",
+      "Setting inputs...\n",
+      "\n",
+      "2022-07-21 10:56:57 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.properties_in[0.0].pressure\n",
+      "2022-07-21 10:56:57 [WARNING] idaes.core.util.scaling: Trying to get scaling factor for unnamed expr \n",
```

Comment:
We need to fix this (or hide it) - my first guess is that the property package in use is not using named expressions in the `get_material_flow_terms`, etc.